src/courses/mappers/16.md

```diff
@@ -0,0 +1,137 @@
+---
+order: 16
+next: 17.md
+title: More Practice
+author: Charles Hu
+---
+
+The following are some practice source data sets to create mappers with.
```

Comment:
I would add some more helper stuff here.  Remember you can't just yeet the students at this and expect them to have success.  For ex. link to or just paste in the template hdf mapping.

You'll also want to prompt the student to remember what the mapping process looks like - make it a question/answer drop down to provide the break down data / annotate source -> correlate schemas / handwrite original -> actually write mapper process.

Reply:
Addressed.